### DISCUSSION TITLE
Requesting Postman Collection for API Testing

### DISCUSSION THREAD

**Discussion by priya23595** [Sunbird-cQube/community#782] [Q&A]
If anyone already has a Postman collection available for the Digital Green project

**Comment by RNarmatha** [comment]
Hi @priya23595 - You are looking postman collection for cQube ?

**Reply by priya23595** [reply]
yes

**Comment by RNarmatha** [comment]
Hi @priya23595 - Here the cQube latest [postman](https://drive.google.com/drive/folders/1_vDU57Q6NrEY-C22Sv4sy56ZAif3uHvT?usp=drive_link) collection
